python/visualization.py

```diff
@@ -988,27 +989,160 @@ def plot2D(
     return ax
 
 
-def plot3D(sim: Simulation):
-    from mayavi import mlab
+def plot3D(sim):
+    from vispy.scene.visuals import Box, Mesh
+    from vispy.scene import SceneCanvas, transforms
+    from skimage.measure import marching_cubes
```

Comment:
On my local machine running Python 3.8 and scikit-image 0.16.2, I receive this import error:
```
ImportError: cannot import name 'marching_cubes' from 'skimage.measure'
```
This is because in [scikit-image 0.16.x](https://scikit-image.org/docs/0.16.x/api/api.html), the marching cube function appears as `marching_cubes_lewiner` and `marching_cubes_classic`. The `marching_cubes` function was added in [0.17.x](https://scikit-image.org/docs/0.17.x/api/skimage.measure.html#marching-cubes) with default `lewiner` algorithm.

Thus, in order to be backward compatible, this line should provide for exception handling:
```py
try:
   from skimage.measure import marching_cubes
except:
   from skimage.measure import marching_cubes_lewiner as marching_cubes
```

Reply:
Thanks for catching that, I missed this accidentally.